gateway/src/db.ts

```diff
@@ -132,7 +132,9 @@ export class LimitDbD1 extends LimitDb {
         `\
 INSERT INTO spend (entityType, entityId, scope, scopeInterval, spendingLimit, spend)
 VALUES ${sqlValues.join(', ')}
-ON CONFLICT DO UPDATE SET spend = spend + EXCLUDED.spend
+ON CONFLICT DO UPDATE SET
+  spend = spend + EXCLUDED.spend,
+  spendingLimit = EXCLUDED.spendingLimit
```

Comment:
no, that's not correct, we shouldn't update `spendingLimit` here too.

Reply:
Removed